src/Data/Either/Combinators.hs

```diff
@@ -313,10 +313,10 @@ rightToMaybe = either (const Nothing) Just
 
 -- | Maybe produce a 'Left', otherwise produce a 'Right'.
 --
--- >>> maybeToRight "default" (Just 12)
+-- >>> maybeTLeft "default" (Just 12)
```

Comment:
s/`maybeTLeft`/`maybeToLeft`/

Reply:
Ah, thanks for catching the typo :stuck_out_tongue: